Src/AmrCore/AMReX_Interp_1D_C.H

```diff
@@ -105,6 +106,35 @@ face_linear_interp_x (int i, int /*j*/, int /*k*/, int n, Array4<T> const& fine,
     }
 }
 
+template <typename T>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+void ccquartic_interp (int i, int /*j*/, int /*k*/, int n,
+                       IntVect const& /* ratio */,
+                       Array4<T const> const& crse,
+                       Array4<T>       const& fine ) noexcept
+
+{
+    // Note: there are asserts in CellConservativeQuartic::interp()
+    //       to check whether ratio is all equal to 2.
+
+    const Array1D<Real, -2, 2> cL = { -0.01171875,  0.0859375, 0.5, -0.0859375, 0.01171875 };
```

Comment:
```suggestion
    constexpr Array1D<Real, -2, 2> cL = { -0.01171875_rt,  0.0859375_rt, 0.5_rt, -0.0859375_rt, 0.01171875_rt };
```
Make similar changes in 2d and 3d.